prost-derive/src/lib.rs

```diff
@@ -412,23 +414,21 @@ fn try_oneof(input: TokenStream) -> Result<TokenStream, Error> {
         }
     }
 
-    let mut tags = fields
+    if fields.iter().any(|(_, field)| field.tags().len() > 1) {
+        panic!("variant with multiple tags"); // Not clear if this is possible, but good to be safe
```

Comment:
This error message is a regression, right? The old message was: "invalid oneof variant {}::{}: oneof variants may only have a single tag"

Reply:
i can re-expand this, sure